client/src/components/RankingQueryResultItem.jsx

```diff
@@ -0,0 +1,74 @@
+import PropTypes from 'prop-types';
+
+function RankingQueryResultItem({ queryValue, img, name, description, score, reviews, rank }) {

```

Comment:
Ez egy Recipe tulajdonképpen. Nem lehet benne felhaszánlni egy máshol már megírt recept kinézetet?

Reply:
A jövőben lehet hogy igen, viszont a szimpla recipe komponens egyenlőre ide nem elég, mert adatok is kellhetnek(rangsor, reviewk száma ...stb). Továbbá nem csak recipe kártyák lesznek, hanem user-ek és nem biztos hogy kártya formában akarjuk megmutatni az adatokat. Lehet hogy egy táblázat kellene ide. Azért csináltam egy önálló komponenst erre, hogy ne korlátozzon be egy már létező komponens. 

Én ebben a sprintben ezt így hagynám.

Reply:
okés, azt meg tudjuk csinálni hogy optional propként adunk meg rank-ot meg score-t a recipe cardnak és a recipe kártya ezt érzékeli és máshogy rendereli ki magát, ha ezeket nem kapja akkor pedig kisebb méretben jelenik meg. Egyébként ha azt látod hogy többet nyerünk azzal hogy ha ez így bemegy akkor nekem ok.